lib/place_mark.rb

```diff
@@ -12,9 +12,7 @@ def execute(player:, x:, y:)
     if check_dup?(x, y)
       { board: @board }
     else
-      @board.each do |_b|
-        @board[x][y] = player
-      end
+      @board.each { @board[x][y] = player }
```

Comment:
I'm not sure if I'm understanding this wrong but as you don't change what x and y are in the .each loop, is this not just assigning player to @board[x][y] 3 times?

Would @board[x][y] = player do the same thing?
(again I may of totally got the wrong end of the stick here)